### PR TITLE
Bump OpenSearch released version to 2.8.0

### PR DESCRIPTION
**Pull request by reta**
### Description
Bump OpenSearch released version to 2.8.0

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.1.0', '1.2.4', '1.3.4', '2.2.0', '2.4.0', '2.6.0']
+        version: ['1.1.0', '1.2.4', '1.3.4', '2.2.0', '2.4.0', '2.6.0', '2.6.0']
```
Comment (dblock) [inline .github/workflows/test.yml:66]: You mean 2.8.0? ;)

Comment (reta) [reply .github/workflows/test.yml:66]: Surely, thanks @dblock !

File: .github/actions/start-opensearch/action.yml [hunk 2]
```diff
@@ -69,6 +69,7 @@ runs:
             exit 0
           fi
           echo '=====> waiting...'
+          cat $(cygpath -alw $OPENSEARCH_HOME/logs/opensearch.log)
```
Comment (reta) [inline .github/actions/start-opensearch/action.yml:72]: Debugging Windows GA failures ...

**Review by VachaShah — APPROVED** [review]

File: .github/actions/start-opensearch/action.yml [hunk 3]
```diff
@@ -25,6 +25,7 @@ runs:
           brew install -q coreutils
         fi
         OPENSEARCH_HOME=$(realpath ./opensearch-*)
+        OPENSEARCH_JAVA_OPTS="-Djava.net.preferIPv4Stack=true"
```
Comment (reta) [inline .github/actions/start-opensearch/action.yml:28]: Getting around of :+1: 
```
	at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method) ~[?:?]
	at java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:930) ~[?:?]
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1543) ~[?:?]
	at java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848) ~[?:?]
	at java.net.InetAddress.getAllByName0(InetAddress.java:1533) ~[?:?]
	at java.net.InetAddress.getAllByName(InetAddress.java:1386) ~[?:?]
	at java.net.InetAddress.getAllByName(InetAddress.java:1307) ~[?:?]
	at org.opensearch.transport.TcpTransport.parse(TcpTransport.java:615) ~[opensearch-2.8.0.jar:2.8.0]
	at org.opensearch.transport.TcpTransport.addressesFromString(TcpTransport.java:557) ~[opensearch-2.8.0.jar:2.8.0]
	at org.opensearch.transport.TransportService.addressesFromString(TransportService.java:1119) ~[opensearch-2.8.0.jar:2.8.0]
	at org.opensearch.discovery.SeedHostsResolver.lambda$resolveHostsLists$0(SeedHostsResolver.java:182) ~[opensearch-2.8.0.jar:2.8.0]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:747) ~[opensearch-2.8.0.jar:2.8.0]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:829) [?:?]
```

File: .github/workflows/test.yml (hunk 1 above)
Comment (dblock) [reply .github/workflows/test.yml]: I don't just blinding merge all your PRs @reta :)

Comment (reta) [reply .github/workflows/test.yml]: :smile: thanks @dblock !

**Pull request merged**